doc/source/whatsnew/v0.20.0.txt

```diff
@@ -31,7 +31,15 @@ Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
 
+ - The ``dtype`` keyword argument in the :func:`read_csv` function for specifying the types of parsed columns
+ is now supported with the ``'python'`` engine.  See the :ref:`io docs <io.dtypes>` for more information.
 
+.. ipython:: python
+
+   from io import StringIO
```

Comment:
`from pandas.compat import StringIO` is imported in a suppressed ipython code block at the top of the file, so you can just use `StringIO`